### PR TITLE
Fix #181, add exit command to interactive shell

### LINKED ISSUE #181: arr.ai shell: /exit command
- [ ] Update reference docs
- [ ] Update shell tutorial

### PR DESCRIPTION
**Pull request by nofun97**
Fixes #181 .

Changes proposed in this pull request:
- Add exit command to interactive shell

Checklist:
- [x] Made corresponding changes to the documentation

### CODE REVIEW

File: cmd/arrai/shell_cmd.go [hunk 1]
```diff
@@ -71,8 +72,19 @@ func (uc *unsetCmd) process(line string, shellData *shellInstance) error {
...
 }
 
+type exitCommand struct{}
+
+func (*exitCommand) name() string {
+	return "exit"
+}
+
+func (ec *exitCommand) process(_ string, _ *shellInstance) error {
+	os.Exit(0)
```
Comment (marcelocantos) [inline cmd/arrai/shell_cmd.go:82]: Would prefer a less brutal approach. Return an error code triggering a graceful shutdown.

File: README.md [hunk 2]
```diff
@@ -133,6 +137,17 @@ from the global scope.
 pew INFO
 ```
 
+#### exit
```
[the remark this answers is not in the document]
Comment (nofun97) [reply README.md]: added

File: cmd/arrai/shell_cmd.go (hunk 1 above)
Comment (nofun97) [reply cmd/arrai/shell_cmd.go]: returned a custom error

File: README.md [hunk 3]
```diff
@@ -133,6 +137,17 @@ from the global scope.
...
 
+#### `/exit`
+
+Usage:
+
+```bash
+@> /exit
+```
+
+The `exit` command exits the interactive shell. Alternatively, you can press
```
Comment (marcelocantos) [inline README.md:148]:
```suggestion
The `/exit` command exits the interactive shell. Alternatively, you can press
```

File: README.md [hunk 4]
```diff
@@ -51,7 +51,11 @@ split      sub        title      upper
 'HELLO'
 ```
 
-Ctrl+D to exit.
+Ctrl+D to exit. or use the `exit` command.
```
Comment (marcelocantos) [inline README.md:54]:
```suggestion
Ctrl+D to exit. or use the `/exit` command.
```
Follow suit with the other commands.

**Pull request merged**